### PR TITLE
access package assignment removal

### PR DESCRIPTION
**Pull request by markwahl-msft**
Add example to the accessPackageAssignmentRequest article from the tutorial that shows how an admin can request to remove an access package assignment.

#12619

### CODE REVIEW

File: api-reference/beta/api/accesspackage-delete.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Namespace: microsoft.graph
 
 Delete an [accessPackage](../resources/accesspackage.md) object.
 
+Note that an access package cannot be removed if there are still assignments to that access package.  In order to remove assignments, [query the assignments](accesspackageassignment-list.md) with a filter to indicate the specific access package, such as: `$filter=accessPackage/id eq 'a914b616-e04e-476b-aa37-91038f0b165b'`, and then [create a new accessPackageAssignmentRequest](accesspackageassignmentrequest-post.md) in which the value of the **requestType** property is `AdminRemove`, to remove each assignment that is still delivered.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackage-delete.md:18]:
```suggestion
You cannot delete an access package if it has an **accessPackageAssignment**. To delete the access package, first remove the assignments. For more, see [Remove an assignment](accesspackageassignmentrequest-post.md#example-4-remove-an-assignment).
```

File: api-reference/beta/api/accesspackageassignmentrequest-post.md [hunk 2]
```diff
@@ -333,6 +333,55 @@ Content-type: application/json
 }
 ```
 
+### Example 4: Remove an assignment
+#### Request
+
+The following example shows how to remove an assignment.
+
+
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackageassignmentrequest-post.md:341]:
```suggestion
### Example 4: Remove an assignment

To remove assignments, create a new accessPackageAssignmentRequest object with the following settings:
+ The value of the **requestType** property set to `AdminRemove`.
+ In the accessPackageAssignment property, include a list of identifiers of the accessPackageAssignment objects to delete.

#### Request
```

**Review by FaithOmbongi — APPROVED** [review]

File: api-reference/beta/api/accesspackageassignmentrequest-post.md (hunk 2 above)
Comment (markwahl-msft) [reply api-reference/beta/api/accesspackageassignmentrequest-post.md]: added

File: api-reference/beta/api/accesspackage-delete.md (hunk 1 above)
Comment (markwahl-msft) [reply api-reference/beta/api/accesspackage-delete.md]: updated paragraph.

**Pull request merged**